packaging/installer/kickstart.sh

```diff
@@ -877,6 +879,22 @@ try_package_install() {
       uninstall_subcmd="remove"
       INSTALL_TYPE="binpkg-rpm"
       ;;
+    rocky)
+      if command -v dnf > /dev/null; then
+        pm_cmd="dnf"
+        repo_subcmd="makecache"
+      else
+        pm_cmd="yum"
+      fi
+      repo_prefix="el/8"
+      pkg_type="rpm"
+      pkg_suffix=".noarch"
+      pkg_vsep="-"
+      pkg_install_opts="${interactive_opts}"
+      repo_update_opts="${interactive_opts}"
+      uninstall_subcmd="remove"
+      INSTALL_TYPE="binpkg-rpm"
+      ;;
```

Comment:
If we’re setting `DISTRO_COMPAT_NAME` correctly, this case should not be needed (IOW, the variables should be identical between Rocky and CentOS, so we set `DISTRO_COMPAT_NAME` to `centos` and just use that case in this case statement).

Reply:
The only difference is `repo_prefix="el/${SYSVERSION}"` part (assuming it is always 8 for rocky, could be 7 for centos).

Reply:
for rockylinux we need to use `el/8` otherwise it won't work